src/components/Text/Text.st.css

```diff
@@ -23,6 +24,11 @@
       @default color-5
     */
     MainTextColor: --overridable;
+    /*
+      The color to use instead of default color for secondary text
+      @default color-4
+    */
+    SecondaryTextColor: --overridable;
```

Comment:
We don't need 2 different vars here, we could just use the `MainTextColor` to override the primary and secondary priority.

So you could just update the comments: 
/*
      The color of the text instead of the default color **set by priority**
      @default for primary priority: color-5,  for secondary priority: color-4
*/
 MainTextColor: --overridable;
 
 and use it as follow:

 `
.root:priority(primary){
    color: color(fallback(value(MainTextColor), value(DefaultTextColor)));
}

.root:priority(secondary){
    color: color(fallback(value(MainTextColor), value(DefaultSecondaryTextColor)));
}`

Reply:
updated